app/client/apiconsole.ts

```diff
@@ -293,6 +293,25 @@ function initialize(appModel: AppModel) {
 
 function requestInterceptor(request: SwaggerUI.Request) {
   delete request.headers.Authorization;
+  const url = new URL(request.url);
+  // Swagger will use this request interceptor for several kinds of
+  // requests, such as requesting the API YAML spec from Github:
+  //
+  //      Function to intercept remote definition, "Try it out",
+  //      and OAuth 2.0 requests.
+  //
+  //    https://swagger.io/docs/open-source-tools/swagger-ui/usage/configuration/
+  //
+  // We want to ensure that only "Try it out" requests have XHR, so
+  // that they pass a same origin request, even if they're not GET,
+  // HEAD, or OPTIONS. "Try it out" requests are the requests to the
+  // same origin.
+  if (url.origin === window.origin) {
```

Comment:
Please be sure to verify this works once it reaches staging or prod, since there it's common, I think, to have the window on a URL like `https://<org-subdomain>.getgrist.com/apiconsole` and the request going to `https://api.getgrist.com/...`.

Reply:
I just checked, and it looks like the relevant request, the one initiated by the Swagger API console, does indeed go back to the same origin.
![image](https://github.com/user-attachments/assets/d2ec0d23-99fe-4358-b0f7-ccae03a71f67)

Reply:
^ this is a good point. I just double-checked in prod, and the api console sends requests to the same domain.